docs/logging.rst

```diff
@@ -194,16 +194,16 @@ line, following the `logfmt <https://brandur.org/logfmt>`_ idea. e.g.::
     so we define it as:
 
     * keys: any string, except:
-        - ' ' is replaced by '_'
-        - '"' and '=' are replaced by ''
+        - ' ', '.', and '=' are replaced by '_'
+        - '"' is replaced by ''
```

Comment:
It might readability to double backtick escape the double ASCII quotation marks:
```
'``"``' is replaced by ''
```